Makefile

```diff
@@ -32,7 +32,7 @@ run-rsp:
 	cd ./loss-detection-app && \
 	docker-compose -f docker-compose.rsp.yml up -d
 
-run-full: run-base run-vap run-rsp
+run-full: models run-base run-vap run-rsp
```

Comment:
run-vap should depends models; and once that dependency is in place, then you don't need models here as dependency